backend/main/chapters/c09_combining_booleans.py

```diff
@@ -0,0 +1,638 @@
+# flake8: NOQA E501
+
+from typing import List
+
+from main.exercises import assert_equal
+from main.text import ExerciseStep, Page, VerbatimStep
+from main.utils import returns_stdout
+
+
+class IntroducingOr(Page):
+    class InputAliceBob(VerbatimStep):
+        """
+We learned about *booleans* (`True` and `False`) when we introduced If statements.
+We also learned about comparison operators `==`, `<`, `>`, `<=`, `>=` which return booleans.
+Now we want to combine booleans to check for more complex conditions.
+
+Here's a simple example: imagine you have two friends, Alice and Bob.
+The function below accepts one parameter, `name`, and checks if the person with the given name is among your friends.
+Below its definition, we test the function on three different inputs with `assert_equal` from the previous chapter.
+After that, we have an example program that uses this function interactively.
+Type and run the code in the editor:
```

Comment:
As discussed in #35, I think this is an example of when we would write "copy" rather than "type". This snippet is long and doesn't have any unfamiliar concepts yet.

Reply:
Got it. Changing "type" to "copy".